guides/01_getting-started/01_quickstart.md

```diff
@@ -1,136 +1,114 @@
+To learn more about sharing your demo, read our dedicated guide on [sharing your Gradio application](/guides/01_getting-started/03_sharing-your-app.md).
 
-Here's a toy example:
 
-$code_chatinterface_random_response
-$demo_chatinterface_random_response
+## An Overview of Gradio
 
-You can [read more about `gr.ChatInterface` here](https://gradio.app/guides/creating-a-chatbot-fast).
+So far, we've been discussing the `Interface` class, which is a high-level class that lets to build demos quickly with Gradio. But what else does Gradio do?
 
-## Blocks: More Flexibility and Control
+### Chatbots with `gr.ChatInterface`
 
-Gradio offers two approaches to build apps:
+Gradio includes another high-level class, `gr.ChatInterface`, which is specifically designed to create Chatbot UIs. Similar to `Interface`, you supply a function and Gradio creates a fully working Chatbot UI. If you're interested in creating a chatbot, you can jump straight [our dedicated guide on `gr.ChatInterface`](/guides/creating-a-chatbot-fast).
 
-1\. **Interface** and **ChatInterface**, which provide a high-level abstraction for creating demos that we've been discussing so far.
+### Custom Demos with `gr.Blocks`
 
-2\. **Blocks**, a low-level API for designing web apps with more flexible layouts and data flows. Blocks allows you to do things like feature multiple data flows and demos, control where components appear on the page, handle complex data flows (e.g. outputs can serve as inputs to other functions), and update properties/visibility of components based on user interaction — still all in Python. If this customizability is what you need, try `Blocks` instead!
+Gradio also offers a low-level approach for designing web apps with more flexible layouts and data flows with the `gr.Blocks` class. Blocks allows you to do things like control where components appear on the page, handle complex data flows (e.g. outputs can serve as inputs to other functions), and update properties/visibility of components based on user interaction — still all in Python. 
 
-## Hello, Blocks
+You can build very custom and complex applications using `gr.Blocks()`. For example, the popular image generation [Automatic1111 Web UI](https://github.com/AUTOMATIC1111/stable-diffusion-webui) is built using Gradio Blocks. We dive deeper into the `gr.Blocks` on our series on [building with Blocks](/guides/03_building-with-blocks/01_blocks-and-event-listeners.md).
 
-Let's take a look at a simple example. Note how the API here differs from `Interface`.
 
-$code_hello_blocks
-$demo_hello_blocks
+### The Gradio Python & JavaScript Ecosystem
 
-Things to note:
+So far, we've been discussing the core `gradio` Python library. But Gradio is actually so much more! Its an entire ecosystem of Python and JavaScript libraries that let you build machine learning applications, or query them programmatically, in Python or JavaScript. Here are other related parts of the Gradio ecosystem:
```

Comment:
```suggestion
That's the gist of the core `gradio` Python library, but Gradio is actually so much more! Its an entire ecosystem of Python and JavaScript libraries that let you build machine learning applications, or query them programmatically, in Python or JavaScript. Here are other related parts of the Gradio ecosystem:
```

Reply:
You've already used "So far, we've been discussing " earlier.

Reply:
Also sometimes you put gradio in a code block like this: `gradio` and sometimes you capitalize it: Gradio 

I guess maybe you mean when it's the core python library vs Gradio ecosystem? Even so, that hasn't been consistent

Reply:
Haha ok good point